frontend/src/main/scala/bloop/engine/tasks/compilation/CompileGraph.scala

```diff
@@ -453,7 +453,7 @@ object CompileGraph {
                           .+=(newProducts.readOnlyClassesDir.toFile -> newResult)
                       case (p, ResultBundle(f: Compiler.Result.Failed, _, _, _)) =>
                         f.bestEffortProducts.foreach {
-                          case BestEffortProducts(products, _) =>
+                          case BestEffortProducts(products, _, _) =>
```

Comment:
Btw. I just noticed `isBestEffortDep` is always false?

Reply:
Thank you for catching it, it must have become redundant in one of the iterations of the first PR and I didn't notice that before. Removed now